package.json

```diff
@@ -52,5 +51,9 @@
     "selector",
     "datetime",
     "ui"
-  ]
+  ],
+  "engines": {
+    "npm": "~2.13.0",
+    "node": "~0.12.0"
```

Comment:
I'm using node 4.0.0 right now, so we should probably change this to `>` or use a range or something..